.amethyst.sample.yml

```
@@ -1,42 +1,280 @@
+  # - widescreen-tall
+  # - widescreen-tall-right
+  # - bsp
+
+#	First mod (default option + shift).
 mod1:
-    - option
-    - shift
-    - command
+  - option
+  - shift
+  # - control
+  # - command
+
+# Second mod (default option + shift + control).
 mod2:
-    - option
-    - shift
-    - control
-    - command
+  - option
+  - shift
+  - control
+  # - command
+
+# Commands:
+# special key values
+# "space"
+# "enter"
+# "up"
+# "right"
+# "down"
+# "left"
+
+# Move to the next layout in the list.
+cycle-layout:
+  mod: mod1
+  key: "space"
+
+# Move to the previous layout in the list.
+cycle-layout-backward:
+  mod: mod2
+  key: "space"
+
+# Shrink the main pane by a percentage of the screen dimension as defined by window-resize-step. Note that not all layouts respond to this command.
+shrink-main:
+  mod: mod1
+  key: h
+
+# Expand the main pane by a percentage of the screen dimension as defined by window-resize-step. Note that not all layouts respond to this command.
+expand-main:
+  mod: mod1
+  key: l
+
+# Increase the number of windows in the main pane. Note that not all layouts respond to this command.
+increase-main:
+  mod: mod1
+  key: ","
+
+# Decrease the number of windows in the main pane. Note that not all layouts respond to this command.
+decrease-main:
+  mod: mod1
+  key: "."
+
+# General purpose command for custom layouts. Functionality is layout-dependent.
+# command1:
+#   mod: <NONE>
+#   key: <NONE>
+# General purpose command for custom layouts. Functionality is layout-dependent.
+# command2:
+#   mod: <NONE>
+#   key: <NONE>
+# General purpose command for custom layouts. Functionality is layout-dependent.
+# command3:
+#   mod: <NONE>
+#   key: <NONE>
+# General purpose command for custom layouts. Functionality is layout-dependent.
+# command4:
+#   mod: <NONE>
+#   key: <NONE>
+
+# Focus the next window in the list going counter-clockwise.
+focus-ccw:
+  mod: mod1
+  key: j
+
+# Focus the next window in the list going clockwise.
+focus-cw:
+  mod: mod1
+  key: k
+
+# Focus the main window in the list.
+focus-main:
+  mod: mod1
+  key: m
+
+# Focus the next screen in the list going counter-clockwise.
+focus-screen-ccw:
+  mod: mod1
+  key: p
+
+# Focus the next screen in the list going clockwise.
+focus-screen-cw:
+  mod: mod1
+  key: n
+
+# Move the currently focused window onto the next screen in the list going counter-clockwise.
+swap-screen-ccw:
+  mod: mod2
+  key: h
+
+# Move the currently focused window onto the next screen in the list going clockwise.
+swap-screen-cw:
+  mod: mod1
+  key: l
+
+# Swap the position of the currently focused window with the next window in the list going counter-clockwise.
+swap-ccw:
+  mod: mod2
+  key: j
+
+# Swap the position of the currently focused window with the next window in the list going clockwise.
+swap-cw:
+  mod: mod2
+  key: k
+
+# Swap the position of the currently focused window with the main window in the list.
+swap-main:
+  mod: mod1
+  key: "enter"
```

Based on suggestion above
```suggestion
  key: enter
```